Sources/tart/Commands/IP.swift

```diff
@@ -13,46 +19,40 @@ struct IP: AsyncParsableCommand {
   @Option(help: "Number of seconds to wait for a potential VM booting")
   var wait: UInt16 = 0
 
+  @Option(help: ArgumentHelp("Strategy for resolving IP adress: dhcp or arp",
+                             discussion: """
+                             By default, Tart is looking up and parsing DHCP lease file to determine aт IP for the VM.\n
+                             This method is fast and the most reliable but only returns local IP adresses.\n
+                             Alternatively, Tart can call external `arp` executable and parse it's output.\n
+                             In case of enabled Bridged Networking this method will return an IP adress on the network interface used for Bridged Networking.\n
```

Comment:
```suggestion
                             In case of enabled Bridged Networking this method will return VM's IP address on the network interface used for Bridged Networking.\n
```